blockchain/blockchain.go

```diff
@@ -1,12 +1,16 @@
-//go:generate abigen --abi ../resources/Agent.abi --pkg blockchain --type Agent --out agent.go
+//go:generate abigen --abi ../resources/blockchain/node_modules/singularitynet-platform-contracts/abi/Agent.json --pkg blockchain --type Agent --out agent.go
 
 package blockchain
 
 import (
 	"bytes"
+	"context"
 	"crypto/ecdsa"
 	"encoding/hex"
 	"encoding/json"
+
```

Comment:
did you manually do this, or is your goland still doing something weird? should be in the first stanza, not separated (and should happen automatically)

Reply:
this was the result of running goimports, but when i move it up below crypto/ecdsa and rerun, it stays there... not sure what's going on